sonaranalyzer-dotnet/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The NuGet config file is needed when restore runs on Azure Devops builds. The [nuget.org](nuget.org) source is not added by default when using `nuget restore` or `dotnet build/restore` commands.

Reply:
like we discussed offline, `sonar-dotnet\sonaranalyzer-dotnet\tests\SonarAnalyzer.UnitTest\NuGet.config` should be harmonized with this one